prepare-release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+if [ "$1" == "" ]; then
+    echo "Expected usage:"
+    echo "./prepare-release.sh [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>] | from-git]"
+    exit 1
+fi
+
+set -euxo pipefail
+
+npm_version="$*"
+echo "Preparing release '$npm_version'"
+
+# Find all package.json files in subdirectories, except node_modules, examples and irma-frontend
+standalone_packages=`find . -name "package.json" -not -path "*/node_modules/*" -not -path "*/examples/*" -not -path "*/irma-frontend/*"`
```

Comment:
Why does `irma-frontend` get special treatment, with its own script?

Reply:
`irma-frontend` has some of the other packages as dependencies. This means that at first the other packages must be updated and only then we can update this package. If I'd do it in one prepare script, then I should `npm link` (which creates local symlinks) all other dependencies to `irma-frontend` first, because then the new version number is not published yet. This is kind of ugly. Then I'm not fully sure that all dependency version numbers are updated correctly. 

By separating the scripts, there is a natural checking point. Then you can first prepare the standalone packages, check them and release them if anything is correct and then prepare `irma-frontend`. It would be possible to merge the `release.sh` and `release-irma-frontend.sh` scripts, but then I'll combine a prepare and a release script. I didn't like that either. Now I think about it, it would be more clear to re-name the `release-irma-frontend.sh` to `prepare-irma-frontend.sh` to make more clear this is a preparing script.